src/events/messageDelete.ts

```diff
@@ -41,21 +34,7 @@ async function log(message: Message, client: Bot) {
     )
         executor = lastEntry.executor;
     client.lastLoggedDeletion = lastEntry;
```

Comment:
this wont work now that the bot is supposed to work on multiple servers
refer to my review on `/src/events/ready.ts` for more info